docs/content/Coding Milestones/PoC2023q1/where-resolver-subs/where-resolver-1-build-kubestellar.md

```diff
@@ -1,5 +1,9 @@
 <!--where-resolver-1-build-kubestellar-start-->
 ```shell
+make imports
+make codegen
+make crds
+make update-contextual-logging
```

Comment:
These do not belong here.
These affect files that are maintained in git.
Developers making relevant changes are responsible getting these done.

Reply:
yes and no. 1) Is there anywhere in the documentation that instructs contributors to take those steps before commiting their changes? maybe I forgot that's in the instructions1 or didn't see it? I do it, but that's because of discussions with our team. 2) does it hurt?

Reply:
We lack proper documentation how to build.  The proposed solution in #821 calls for that.

Having these commands here does not cause the execution to fail, but it does lead human readers in the wrong direction.